scripts/001-binutils-PPU.sh

```diff
@@ -3,41 +3,13 @@
 
 BINUTILS="binutils-2.22"
 
-if [ -d ../src/${BINUTILS}-PS3 ]; then
-  BINUTILS="../src/${BINUTILS}-PS3"
-elif [ ! -d ${BINUTILS} ]; then
-
-  ## Download the source code.
-  if [ ! -f ${BINUTILS}.tar.bz2 ]; then wget --continue https://ftp.gnu.org/gnu/binutils/${BINUTILS}.tar.bz2; fi
-
-  ## Unpack the source code.
-  tar xfvj ${BINUTILS}.tar.bz2
-
-  ## Patch the source code.
-  cat ../patches/${BINUTILS}-PS3.patch | patch -p1 -d ${BINUTILS}
-fi
-
-if [ -d ../src/config ]; then
-    CONFIG=../src/config
-
-## Download an up-to-date config.guess and config.sub
-else
-    CONFIG=.
-fi
-
-if [ ! -f $CONFIG/config.guess ]; then wget --continue https://git.savannah.gnu.org/cgit/config.git/plain/config.guess; fi
-if [ ! -f $CONFIG/config.sub ]; then wget --continue https://git.savannah.gnu.org/cgit/config.git/plain/config.sub; fi
+cp -rf ${PS3DEV_SOURCES}/${BINUTILS} ${BINUTILS}
```

Comment:
Copying the whole thing seems a bit excessive, isn't a symlink enough?

Reply:
Don't want to make the submodule dir dirty, it takes almost no time for me here. Otherwise you'd have to cd into each submodule and git clean -df && git reset -- hard

Reply:
Ok, so how about this:  Instead of copying or making a symlink, just create an empty directory.  Then change `../configure` to `${PS3DEV_SOURCES}/${BINUTILS}/configure`.  We're doing an out-of-tree build anyway...